openfecli/commands/quickrun.py

```diff
@@ -33,10 +33,9 @@ def _format_exception(exception) -> str:
 )
 @click.option(
     'output', '-o', default=None,
-    type=click.Path(dir_okay=False, file_okay=True, writable=True,
```

Comment:
I removed `writable=True` here, since click ignores any further parameters if the file does not exist (and here we require that the file doesn't exist). 

Similarly, we should be able to just use `file_okay=False` instead of our own custom check in `ensure_file_does_not_exist()`